templates/repo/issue/view_content.tmpl

```diff
@@ -106,7 +106,7 @@
 												</button>
 											{{end}}
 										{{end}}
-										<button class="ui primary button">
+										<button id="comment-button" class="ui primary button" disabled>
```

Comment:
In the future, it may not be ideal to have `disabled` here. If we want to follow GitHub by having issue/PR comments survive page reloads, we could just remove this attribute and maybe just determine it **dynamically** when we are initializing a comment in the frontend.

Reply:
This `disabled` could be removed. Instead, call `onContentChanged` explicitly during init in `initSingleCommentEditor`

Reply:
Done, though I had to turn `initSingleCommentEditor()` to be async. I'm not yet familiar with async programming in JS so please let me know if there is a problem.